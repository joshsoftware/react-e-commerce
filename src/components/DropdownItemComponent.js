import React from 'react';
import { DropdownItem } from 'reactstrap';
import PropTypes from 'prop-types';

const DropdownItemComponent = ({ option }) => {
  return (
    <>
      <DropdownItem>{option}</DropdownItem>
      <DropdownItem divider />
    </>
  );
};

export default DropdownItemComponent;

DropdownItemComponent.propTypes = {
  option: PropTypes.string.isRequired
};