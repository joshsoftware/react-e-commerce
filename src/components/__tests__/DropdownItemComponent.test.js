import DropdownItemComponent from '../DropdownItemComponent';
import { render } from '@testing-library/react';
import React from 'react';

describe('DropdownItem Component', () => {
  it('Must return Dropdown Item', () => {
    const option = 'Options';
    const { asFragment } = render(<DropdownItemComponent option={option} />);

    expect(asFragment()).toMatchSnapshot();
  });
});
