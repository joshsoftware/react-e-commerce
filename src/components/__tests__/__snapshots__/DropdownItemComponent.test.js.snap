// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DropdownItem Component Must return Dropdown Item 1`] = `
<DocumentFragment>
  <button
    class="dropdown-item"
    role="menuitem"
    tabindex="0"
    type="button"
  >
    Options
  </button>
  <div
    class="dropdown-divider"
    tabindex="-1"
  />
</DocumentFragment>
`;
